app/[lang]/chapters/[slug]/layout.tsx

```diff
@@ -1,13 +1,23 @@
-import Navbar from 'components/Navbar'
+'use client'
+
+import Navbar, { NavbarMobile } from 'components/Navbar'
 import { chapters } from 'content'
+import useMediaQuery from 'hooks/useMediaQuery'
```

Comment:
Please use the following, would better match how we import other hooks:
```
import { useMediaQuery } from 'hooks'
```

Reply:
Addressed in ea822132cc3d13887c40e033b9b5d1605f6ed579
Thanks!